packages/plugins/src/plugins/ts-lint.plugin.ts

```diff
@@ -0,0 +1,42 @@
+import { type PluginResult, Plugin } from '@rehearsal/service';
+import { ESLint } from 'eslint';
+import { debug } from 'debug';
+import { setProcessTTYto } from '../helpers';
+
+const DEBUG_CALLBACK = debug('rehearsal:plugins:no-any');
+
+export class TSLintPlugin extends Plugin {
+  async run(fileName: string): PluginResult {
+    const text = this.service.getFileText(fileName);
+
+    try {
+      const overrideConfig = {
+        plugins: ['@typescript-eslint'],
+        extends: ['plugin:@typescript-eslint/recommended'],
+      };
+      const options = {
+        fix: false,
+        overrideConfig,
+        useEslintrc: false,
```

Comment:
We def want to consume the eslintrc (comment)

Reply:
We can also create a curated eslint config which will extend an existing https://eslint.org/docs/latest/user-guide/configuring/configuration-files#extending-configuration-files

eg. rehearsal:recommended

Reply:
Sounds good.

Reply:
For now I hardcoded an essential eslint config, and add a TODO to move it to `rehearsal:recommended`. Will open a PR later to address it.

Reply:
fixed per discussion